### PR TITLE
Fix: #150 yaml formating of grafana dashboards

### PR DESCRIPTION
**Pull request by Zempashi**
Fix: #150

### CODE REVIEW

File: ChangeLog.rst [hunk 1]
```diff
@@ -14,6 +14,8 @@ Bugs fixed
 ----------
 :ghpull:`151` - fix `debug` clause `var` scoping
 
+:ghissue:`150` - Fix yaml formating of grafana dashboards
```
Comment (NicolasT) [inline ChangeLog.rst]: That's the fix, but not the bug fixed :smiley:

```
:ghissue:`150` - fix deployment of Elasticsearch, node and Prometheus Grafana dashboards (:ghpull:`158`)
```

**Review by NicolasT — CHANGES_REQUESTED** [review]
Minor adjustment to ChangeLog entry please, otherwise LGTM.

File: ChangeLog.rst (hunk 1 above)
Comment (Zempashi) [reply ChangeLog.rst]: what ? that's precisely the bug id ?

Comment (NicolasT) [reply ChangeLog.rst]: No, I meant, the text says what the fix is, not what the bug was. I acknowledge the description of #151 above is similar, but that's a bit of an odd duck.

A user wants to know which bug (i.e. something he could observe) is fixed. `YAML formatting of Grafana dashboards` is not a bug.

**Review by NicolasT — APPROVED** [review]
:100:

**Pull request merged**